counsel.el

```diff
@@ -3127,6 +3127,26 @@ otherwise continue prompting for tags."
 ;;;###autoload
 (defalias 'counsel-org-goto #'counsel-outline)
 
+(defcustom counsel-org-goto-all-outline-path-prefix nil
```

Comment:
Just curious: is this too specific a name? Could this user option become useful for a related but slightly different purpose in the future, e.g. outside of "goto-all"?

Reply:
Yeah, that seems a little too specific.

For starters, I think this variable can immediately be used in `counsel-org-agenda-headlines`.

Also, it seems to me that `counsel-org-goto-all` itself should be generalized to other modes supported by `counsel-outline`, and `counsel-org-goto-all` should be a wrapper around this new function. The variable would then be used in the hypothetical `counsel-outline-all` and by proxy, `counsel-emacs-lisp-goto-all`, `counsel-outline-latex-goto-all`, etc.

Given these use cases and the existing variable `counsel-outline-path-separator`, I think a name like `counsel-outline-path-prefix` would be fitting, although it should be made clear that the prefix is only used when outlines from multiple buffers are displayed in one candidate list.

What do you think?

Reply:
SGTM, though it's been a while since I last looked at the implementation of these features, so I could be missing something.

Either way, I don't see anything wrong with making the name of the new user option more general.

Thanks.